design/Exceptions.md

```diff
@@ -0,0 +1,75 @@
+The behavior of every error case in WebGPU is defined by the spec on a
+case-by-case basis. A given error has one of these behaviors:
+
+* Synchronously throws a JS exception.
+* Occurs asynchronously - one of:
+  * Causes a WebGPU object to become internally null. Produces an error log entry.
+  * Causes an operation to no-op. Produces an error log entry.
```

Comment:
ignorable/non-ignorable error semantics?

Reply:
Hm, technically all asynchronous errors are ignorable. If an object is internally-null then it propagates, causing operations using it to become no-ops.